eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -218,6 +270,7 @@ function FindPackageWorkItem($lang, $packageName, $version, $outputCommand = $tr
   $fields += "System.State"
   $fields += "System.AssignedTo"
   $fields += "System.Parent"
+  $fields += "System.Tags"
```

Comment:
What are we getting from the Tags? Is that only to filter out the tests? If so we might want to consider other potential ways to filter these out, like perhaps having some way to close them all so they don't show up in the queries.

Reply:
This is to avoid showing up work items created from release planner app in dev and PPE.

Reply:
Are tags the best way of doing this? These will also start to show up in other query or dashboards that folks might be using directly in DevOps. Can we instead have something that goes in and closes all the dev and PPE created items?

Reply:
Yes, we can mark them as closed by automation pipeline. I will discuss with @maririos and come up with a plan. In the meanwhile, I think we should include current changes so work items created by prepare release script won't attach package WI to incorrect service and product.